packages/fluentui/README.md

```diff
@@ -29,25 +33,28 @@ See [CONTRIBUTING.md](CONTRIBUTING.md) for a step-by-step setup and development
 ## Issues
 
 If you need to [file an issue][3], please note in the title or description that it refers to `@fluentui/react-northstar`.
+List of [active issues][4]
 
 ## Documentation and usage examples
 
 See the doc site at https://aka.ms/fluent-ui.
 
 ## FAQs
 
-### Why the move?
+### What is the future of Fluent UI React Northstar?
 
-We're in the process of converging UI projects at Microsoft under the "Fluent UI" banner to simplify the dev story. In the process, we're making performance improvements and enabling greater interoperability of these components across more places and platforms at Microsoft.
+The future of `@fluentui/react-northstar` is called `@fluentui/react-components`. It builds on top of many base principles that were present in Northstar, streamlines the API and brings significanlty improved performance.
 
-### Why the separate folder?
+### How can we migrate to Fluent UI React Components v9?
 
-These packages remain in a separate folder while we work on converging the build systems and certain other details, but they will eventually move directly under `packages` (or `apps`) next to the other packages.
+Please see the [Migration documentation][5].
 
 ### What's happening with Teams support?
 
-`@fluentui/react-northstar v0.x.x` (formerly `@fluentui/react v0.x.x`) has been the official set of components for Teams. This is still the case and no support is being dropped.
+`@fluentui/react-northstar v0.x.x` (formerly `@fluentui/react v0.x.x`) has been the official set of components for Teams. This is still the case and no support is being dropped. New features will only be added to Fluent UI React Components v9.
 
 [1]: https://github.com/microsoft/fluent-ui-react
 [2]: https://github.com/microsoft/fluentui/blob/master/packages/fluentui/CHANGELOG.md
 [3]: https://github.com/microsoft/fluentui/issues/new/choose
+[4]: https://github.com/microsoft/fluentui/issues?q=is%3Aissue+is%3Aopen+label%3A%22Fluent+UI+react-northstar+%28v0%29%22
+[5]: https://react.fluentui.dev/?path=/docs/concepts-migration-overview--page
```

Comment:
This link will probably break in the near future when Geoff redos the table of contents, I'll leave it you whether you want to just link to the doc site and say "See Migrations Docs on v9 documentation site"

Reply:
we'll fix it then - I think having the links is useful.